docs/learn-balance-transfers.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
@lsaether  did you just update the images? I had updated this in my other PR #913

Reply:
Ah I missed it in that PR. I updated the text as well.